examples/user_guide/10-Transforming_Elements.ipynb

```diff
@@ -298,6 +298,105 @@
+    "        \n",
+    "        # setting flat=False will preserve the matrix shape\n",
+    "        data = element.dimension_values(2, flat=False)[::-1]\n",
+    "        \n",
+    "        if self.p.type_ == \"high-pass\":\n",
+    "            new_data = data - ndimage.gaussian_filter(data, self.p.sigma)\n",
+    "        else:\n",
+    "            new_data = ndimage.gaussian_filter(data, self.p.sigma)\n",
+    "            \n",
+    "        # make an exact copy of all setting and axes etc., just with different data:\n",
+    "        element = element.clone(data=new_data)\n",
+    "        element = element.relabel(element.label + \" ({} filtered)\".format(self.p.type_))\n",
+    "        return element\n",
+    "\n",
+    "stairs_map = hv.HoloMap({sigma: image_filter(stairs_image, sigma=sigma)\n",
+    "                         for sigma in range(0, 12, 1)}, kdims=\"sigma\")\n",
+    "stairs_map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just as in the previous example, it is quite straight-forward to build a HoloMap containing results for different parameter values. Inside the ``_process()`` method, the given parameters can be accessed as ``self.p.<parameter-name>`` (note that ``self.<parameter_name>`` always contains the default value!). Since we did not specify the ``_type`` parameter, it defaulted to \"low-pass\".\n",
+    "\n",
+    "There are some peculiarities when applying operations to two-dimensional elements:\n",
+    "\n",
+    "- Understanding the ``dimension_values()`` method: In principle, one could use ``element.data`` to access the element's data, however, using ``dimension_values()`` means one does not have to worry about different data formats for different elements. The first parameter specifies the dimension to be returned. Values 0 and 1 refer to the horizontal and vertical axis, respectively. For  ``Image`` and ``Raster`` elements, 2 refers to the magnitude, while for ``RGB``, 2, 3 and 4 refer to the red, green and blue channels. Setting ``expanded=False`` yields only the axis, while the default setting ``expanded=True`` returns a value for every pixel. Specifying ``flat=False`` means that the data's matrix shape will be preserved, which is what we need for this kind of filter.\n",
+    "- The image data returned by ``dimension_values()`` must be flipped along the first axis due to conflicting conventions.\n",
+    "- ``Image`` and related classes come with convenient methods to convert between matrix indices and data coordinates and vice versa: ``matrix2sheet()`` and ``sheet2matrix()``. This is useful when searching for features such as peaks.\n",
+    "\n",
+    "A very powerful aspect of operations is the fact that they understand Holoviews data structures. This means it is very straight-forward to apply an operation to every element in a container. As an example, well apply an additional high-pass filter to our HoloMap:"
```

Comment:
we'll

(or "let's")

Reply:
Fixed it (but now I see I made a typo in the commit message...)